.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,2 +1,19 @@
-By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+### What does this PR do / why do we need it?
```

Comment:
nit: Consider more open language like "How does this improve the operators?" instead of "Why do we need it"?

Reply:
Ripped this from k8s. Will do.